Source/Android/jni/MainAndroid.cpp

```diff
@@ -608,7 +609,7 @@ JNIEXPORT jstring JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_GetConfig
   std::string key = GetJString(env, jKey);
   std::string defaultValue = GetJString(env, jDefault);
 
-  ini.Load(File::GetUserPath(D_CONFIG_IDX) + std::string(file));
+  ini.Load(Paths::GetConfigDir() + std::string(file));
```

Comment:
Sorry, I should've probably pointed those out explicitly; but only did the first one (over in `ButtonManager.cpp`) last time...